tests/php/Configuration/Parser/ContentTypesParserTest.php

```diff
@@ -81,9 +81,55 @@ public function testHasConfig(): void
         $this->assertTrue($config['homepage']['viewless']);
         $this->assertTrue($config['homepage']['singleton']);
         $this->assertSame('published', $config['homepage']['default_status']);
-        $this->assertSame('homepage', $config['homepage']['tablename']);
         $this->assertSame('fa-home', $config['homepage']['icon_many']);
         $this->assertSame('fa-home', $config['homepage']['icon_one']);
         $this->assertFalse($config['homepage']['allow_numeric_slugs']);
     }
+
+    public function testInferContentTypeValues(): void
+    {
+        $file = self::getBasePath() . 'minimal_contenttypes.yaml';
+
+        $generalParser = new GeneralParser();
+        $contentTypesParser = new ContentTypesParser($generalParser->parse(), $file);
+        $config = $contentTypesParser->parse();
+
+        $this->assertCount(2, $config);
```

Comment:
magic number

Reply:
There _are_ two in the `minimal_contenttypes.yaml`. What else would I put there?

Reply:
add `NUMBER_OF_CONTENT_TYPES_IN_MINIMAL_FILE` (or something like this) const to this test class :)

Reply:
Done.